sympy/physics/vector/vector.py

```diff
@@ -345,12 +339,20 @@ def __ror__(self, other):
     def __rsub__(self, other):
         return (-1 * self) + other
 
-    def __str__(self, printer=None):
+    def __str__(self, printer=None, order=True):
```

Comment:
It seems that the order is respect to this new index for the reference frame. Why would a user want an index for the frames and want them to be ordered in that way when printing?